From daa5f65e1054091d29e54d16da5f313b3896551b Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Fri, 2 Oct 2020 15:53:50 -0500
Subject: [PATCH 58/62] drm/bridge: sec-dsim: Remove version warning

The Nano TRM shows the value should be 0x1060100, however
in reality it's returning 0x1060200 after initialization
and 0 before initialization.  Since the function works
and the version itself is never actually used, this patch
removes the splat from boot time.

Fixes: PHOENIX8-220

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 drivers/gpu/drm/bridge/sec-dsim.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/bridge/sec-dsim.c b/drivers/gpu/drm/bridge/sec-dsim.c
index 0dfdbe7eb3d6..823a31f3d317 100644
--- a/drivers/gpu/drm/bridge/sec-dsim.c
+++ b/drivers/gpu/drm/bridge/sec-dsim.c
@@ -1863,7 +1863,6 @@ int sec_mipi_dsim_bind(struct device *dev, struct device *master, void *data,
 
 	pm_runtime_get_sync(dev);
 	version = dsim_read(dsim, DSIM_VERSION);
-	WARN_ON(version != pdata->version);
 	pm_runtime_put_sync(dev);
 
 	dev_info(dev, "version number is %#x\n", version);
-- 
2.17.1

